templates/controllers/modals/submissionMetadata/issueEntryTabs.tpl

```diff
@@ -23,6 +23,21 @@
 	{rdelim});
 </script>
 {if not $hideHelp}<p class="pkp_help">{translate key="issue.manage.entryDescription"}</p>{/if}
+<form id='submissionSettingsRevisionForm' class='pkp_form'>
+	{if $submissionSettingsRevisions|@count > 1}
+		<div id='editPreviousRevision'>
+			{fbvFormSection  for="submissionSettingsRevision"}
+				{fbvElement type="select" name="submissionSettingsRevision" id="submissionSettingsRevision" from=$submissionSettingsRevisions translate=false selected=$currentSubmissionSettingsRevision}
```

Comment:
Ah, I have just seen, that also else name is used, so you can leave it... Sorry... :-(